crates/cxx-qt-gen/src/syntax/foreignmod.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) fn foreign_mod_to_foreign_item_types(
         .filter_map(|item| match foreign_item_to_type(item) {
             Ok(Some(value)) => Some(Ok(value)),
             Ok(None) => None,
-            Err(err) => Some(Err(err)),
+            Err(err) => Some(Err(err)), // Hard to cover since foreign_item_to_type never returns error
```

Comment:
You would need `verbatim_to_foreign_type` to have failed i believe?

So eg if you had a `type T = abc` with not `;` at the end this would Err ?

So if you had an input of
```rust
extern "RustQt" {
   type A = B // no semi-colon should fail
}
```

does this work?

Reply:
As discussed this that semi-colon check is never hit, as this happens in syn

- When a `type` is found https://github.com/dtolnay/syn/blob/94d74d52ca8ec4fb46317f261edd678f9231e178/src/item.rs#L1864 it uses `parse_foreign_item_type`
- `parse_foreign_item_type` build a `FlexibleItemType` and returns Verbatim if the type has a colon or type
- When `FlexibleItemType` is run through Parse it requires valid syntax with a `;` at the end otherwise it fails https://github.com/dtolnay/syn/blob/94d74d52ca8ec4fb46317f261edd678f9231e178/src/item.rs#L1149

So if we ever get a Verbatim block that starts with a `type` it must also have a matching `;` at the end, and we only care about the type and the ident in this function, so that whole `input.step` can be removed